collectors/githubreceiver/colconfig.yaml

```diff
@@ -0,0 +1,94 @@
+- op: add
+  path: /spec/config
+  value:
+    extensions:
+      health_check: {}
+
+      pprof:
+        endpoint: 0.0.0.0:1777
+
+      zpages:
+        endpoint: 0.0.0.0:55679
+
+      bearertokenauth/github:
+        token: ${env:GH_PAT}
+
+      # bearertokenauth/gitlab:
```

Comment:
Since this is for the GitHub receiver. Don't necessarily need this GitLab token commented here since it will never be used here.